astropy/wcs/wcs.py

```diff
@@ -977,13 +987,20 @@ def write_dist(num, cpdis):
             image = fits.ImageHDU(cpdis.data, name=str('WCSDVARR'))
             header = image.header
 
-            header[str('CRPIX1')] = (cpdis.crpix[0], 'Coordinate system reference pixel')
-            header[str('CRPIX2')] = (cpdis.crpix[1], 'Coordinate system reference pixel')
-            header[str('CRVAL1')] = (cpdis.crval[0], 'Coordinate system value at reference pixel')
-            header[str('CRVAL2')] = (cpdis.crval[1], 'Coordinate system value at reference pixel')
-            header[str('CDELT1')] = (cpdis.cdelt[0], 'Coordinate increment along axis')
-            header[str('CDELT2')] = (cpdis.cdelt[1], 'Coordinate increment along axis')
-            image.ver = int(hdulist[0].header[str('{0}{1:d}.EXTVER').format(d_kw, num)])
+            header[str('CRPIX1')] = (cpdis.crpix[0],
+                                     'Coordinate system reference pixel')
+            header[str('CRPIX2')] = (cpdis.crpix[1],
+                                     'Coordinate system reference pixel')
+            header[str('CRVAL1')] = (cpdis.crval[0],
+                                     'Coordinate system value at reference pixel')
+            header[str('CRVAL2')] = (cpdis.crval[1],
+                                     'Coordinate system value at reference pixel')
+            header[str('CDELT1')] = (cpdis.cdelt[0],
+                                     'Coordinate increment along axis')
+            header[str('CDELT2')] = (cpdis.cdelt[1],
+                                     'Coordinate increment along axis')
+            image.ver = int(hdulist[0].header[str(
+                '{0}{1:d}.EXTVER').format(d_kw, num)])
```

Comment:
Why are these changes and other similar changes necessary? What makes this section better than the way it was before? It looks like autopep8 was run on this file and it does funny things. To me this is just noise and hard to review. Next time I edit the file my settings may try to revert them. Sorry, but I prefer not to have these type of changes. Is it possible to only do the necessary PEP8 changes?